File: TwoSum/TwoSum_ll.js
function TwoSumll(arr,target){
    let left = 0
    let right = arr.length - 1

    while(left < right){
        let sum = arr[left] + arr[right]
        if(sum < target){
            left+=1;
        }else if(sum > target){
            right-=1
        }else{
            return [left+1, right+1]
        }
        
    }
    return null
}

console.log(TwoSumll([1,2,3,4,5,6,7,11],9))

File: TwoSum/TwoSum_ll.py
def TwoSumll(arr,target):
    left = 0
    right = len(arr) -1
    while left < right:
        curentsum = arr[left] + arr[right]
        if curentsum > target:
            right-=1
        elif curentsum < target:
            left+=1
        else:
            return [left+1,right+1]
        

print(TwoSumll([1,2,3,4,5,6,11,16,233,123,123],9))